### ISSUE TITLE
Re-training a model using weights from a different image set

### ISSUE THREAD

**Issue by matt-sharp** [AlexeyAB/darknet#7950]
I've trained a model using an image set called HRSC2016 (high resolution ship collection) which contains ships in ports.  It achieves a very good level of accuracy on the test set.
I've now obtained some additional data also containing ships in ports which I would like to re-train the model using.
When training the model using the new data as my training set and the validation data from HRSC2016 with the best weights created from HRSC2016 I'm getting no loss error or mAP on my chart.  The best weights file it creates returns a mAP of -nan.

So, there is clearly something wrong with the training.  
Is it something to do with my images and labels or is there something fundamental I'm misunderstanding about how I can re-train using the validation data from the previous image set?

**Comment by matt-sharp** [comment]
Is it only possible to restart training using last weights file and not best weights?

**Comment by lsd1994** [comment]
No, you can retrain with any weights, and you should use HRSC2016+new data to train the model.
